gobrew.go

```diff
@@ -114,31 +114,36 @@ func (gb *GoBrew) Interactive(ask bool) {
 	fmt.Println()
 
 	if currentVersion == NoneVersion {
-		color.Warnln("GO Installed Version", ".......", currentVersion)
+		color.Warnln("🚨 Installed Version", ".......", currentVersion, "⚠️")
```

Comment:
If we use UTF-8 characters, how will this work on remote machines? Or where a different code page is used? Didn't check?

Reply:
didn't check. UTF8 support should be there on most machines. User can always use `gobrew <command>`
This PR only effects the `gobrew` interactive command.

Reply:
OK